nvflare/private/fed/server/job_cmds.py

```diff
@@ -288,11 +288,15 @@ def list_jobs(self, conn: Connection, args: List[str]):
             parser.add_argument(
                 "-a",
                 action="store_true",
-                help="List all jobs, default is filtered to jobs submitted by the same user and limit to last 5 jobs by submit time",
+                help="List all jobs, default is filtered to jobs submitted by the same user",
             )
             parser.add_argument("-n", help="Filter by job name prefix")
             parser.add_argument(
-                "-m", default=5, help="Limit maximum number of jobs returned to the specified number, default is 5"
+                "-m",
+                nargs="?",
+                const=5,
+                type=int,
+                help="Limit maximum number of jobs returned to the specified number, default is 5 if flag is set but there is no integer provided for the specified limit",
```

Comment:
Should try to avoid using nargs="?". This doesn't follow getopts convention and it can be ambiguous. For example, this probably throws an exception,

   list_jobs -m job_id

Reply:
Modified to not use nargs="?".